.github/workflows/ci.yml

```diff
@@ -3,22 +3,30 @@ name: CI
 on: [push]
```

Comment:
maybe this should be run on more than just pushes? other actions run as a cronjob too, see for example 

https://github.com/cyberark/conjur-quickstart/blob/76dbdbf463c3f4405c4c87dc2607eb89b458c25d/.github/workflows/quick_start_flow_test.yml#L5

Reply:
Added the timer. I don't think the request stuff is necessary, it just results in duplicate workflows.